### PR TITLE
Update Backuptool

### PR DESCRIPTION
**Pull request by maf12**
## Description

- Updates backuptool to allow for custom config files with alternative files/folders to backup.
- Changes the backup zip filename to `JELOS_BACKUP.zip` so that potential backups from other CFW do not overwrite it, and vice versa.
- If there is an existing backup when creating a new backup, this will now "archive" the existing one to ensure that it doesn't get overwritten in error.

Fixes # (issue)

N/A

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## How Has This Been Tested Locally?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Run the backup script with no `backuptool.conf`file. The standard files/folders should back up.
- [x] Run the backup script with a `backuptool.conf` file. The custom files/folders should back up.
- [x] Check that the first backup is 'archived' in the backup folder.
- [x] Check that files/folders correctly restore after being backed up. (works for everything except `hypseus` as it's read-only)

**Test Configuration**: Existing backuotool script
* Build OS name and version: N/A
* Docker (Y/N): N
* JELOS Branch: https://github.com/maf12/distribution/tree/backup_update
* Any additional information that may be useful: N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

Note: This PR template is adapted from [embeddedartistry](https://github.com/embeddedartistry/templates/blob/master/oss_docs/PULL_REQUEST_TEMPLATE.md)

### CODE REVIEW

File: packages/jelos/sources/scripts/backuptool [hunk 1]
```diff
@@ -4,26 +4,67 @@
...
+DEFAULT=(
+    /storage/.config/fancontrol.conf
+    /storage/.config/backuptool.conf
+    /storage/.cache/bluetooth/*
+    /storage/.cache/connman*
+    /storage/.config/system/configs/system.cfg
+    /storage/.config/ppsspp/*
+    /storage/.config/retroarch/*
+    /storage/.config/moonlight/*
+    /storage/.config/game/*
```
Comment (maf12) [inline packages/jelos/sources/scripts/backuptool:29]: I added these folders to the default backup

File: packages/jelos/sources/scripts/backuptool [hunk 2]
```diff
@@ -4,26 +4,67 @@
...
-    [ -z "${2}" ] && systemctl start emustation
-  ;;
-esac
+)
+
+if [ -f $BACKUPFILE ]
+then
+    TODAY=`date +%y-%m-%d_%H_%M_%S`
+    ARCHIVEFILENAME="ARCHIVED_JELOS_BACKUP-$TODAY.zip"
+    mv $BACKUPFILE "$BACKUPFOLDER/$ARCHIVEFILENAME"
```
Comment (maf12) [inline packages/jelos/sources/scripts/backuptool:38]: A basic 'archive' to ensure existing backups are not overwritten

File: packages/jelos/sources/scripts/backuptool (hunk 1 above)
Comment (fewtarius) [reply packages/jelos/sources/scripts/backuptool:29]: Did you test a restore?  We want to make sure they're put back in the same place they're archived from.

Comment (maf12) [reply packages/jelos/sources/scripts/backuptool:29]: Good point - I tested the restore on everything except the `game` configs. These are fine except for `hypseus` which won't let you restore as it's read-only.

**Pull request merged**